tests/https/test_unit_http_ssl.py

```diff
@@ -45,9 +45,10 @@
     "https://rsa4096.badssl.com",
     "https://rsa8192.badssl.com",
     "https://sha256.badssl.com",
-    "https://sha384.badssl.com",
-    "https://sha512.badssl.com",
-    # "https://tls-v1-2.badssl.com",
+    # These are temporarily broken, see https://github.com/chromium/badssl.com/issues/501
+    # "https://sha384.badssl.com",
+    # "https://sha512.badssl.com",
+    "https://tls-v1-2.badssl.com",
```

Comment:
I tried loading this and my browser is reporting a secure connection. Is that correct? I would have assumed this site is intended to provide a side with well....bad SSL. :)

Reply:
For `https://tls-v1-2.badssl.com`? Yes, that's supposed to be valid.